src/server/server_family.cc

```diff
@@ -161,6 +168,28 @@ std::string AbslUnparseFlag(const ReplicaOfFlag& flag) {
   return (flag.has_value()) ? absl::StrCat(flag.host, ":", flag.port) : "";
 }
 
+bool AbslParseFlag(std::string_view in, CronExprFlag* flag, std::string* err) {
+  if (in.empty()) {
+    flag->value = std::nullopt;
+    return true;
+  }
+
+  std::string raw_cron_expr{CronExprFlag::cronPrefix};
+  raw_cron_expr += in;
```

Comment:
```suggestion
  std::string raw_cron_expr = absl::StrCat(CronExprFlag::cronPrefix, in);
```
It could save allocation and reads better imho, although the string is smaller than SSO so it's mostly just a good practice